packages/connect/src/api/cardano/api/cardanoComposeTransaction.ts

```diff
@@ -16,16 +18,8 @@ export default class CardanoComposeTransaction extends AbstractMethod<
         const { payload } = this;
 
         // validate incoming parameters
-        validateParams(payload, [
-            { name: 'account', type: 'object', required: true },
-            { name: 'feeLevels', type: 'array' },
-            { name: 'outputs', type: 'array' },
-            { name: 'certificates', type: 'array', allowEmpty: true },
-            { name: 'withdrawals', type: 'array', allowEmpty: true },
-            { name: 'changeAddress', type: 'object', required: true },
-            { name: 'addressParameters', type: 'object', required: true },
-            { name: 'testnet', type: 'boolean' },
-        ]);
+        // TODO: weak assert for compatibility purposes (issue #10841)
+        AssertWeak(CardanoComposeTransactionParamsSchema, payload);
```

Comment:
just to remind myself - it only throws for required properties and possibly for some unknown errors of instance other than InvalidParameter

```
export function AssertWeak<T extends TSchema>(
    schema: T,
    value: unknown,
): asserts value is Static<T> {
    try {
        Assert(schema, value);
    } catch (e) {
        if (e instanceof InvalidParameter) {
            if (e.type === ValueErrorType.ObjectRequiredProperty) {
                // We consider this error to be serious
                throw e;
            }
            console.warn('Method params validation failed', e);
        } else {
            throw e;
        }
    }
}
```

I just trying to figure out what could go wrong if there was a bug in the schema